src/main/java/frc/robot/RobotContainer.java

```diff
@@ -223,21 +224,21 @@ private void configureButtonBindings() {
     // Intake / CDS
     if (intakeForwardCommand != null && outtakeCommand != null) {
       // takes ball in
-      buttons[Constants.RBumper].whileHeld(intakeForwardCommand);
+      buttons[Constants.RTriggerButton].whileHeld(intakeForwardCommand);
       // spits ball out
-      buttons[Constants.RTriggerButton].whileHeld(outtakeCommand);
+      buttons[Constants.RBumper].whileHeld(outtakeCommand);
     }
 
     if (shooterSubsystem != null && shooterHeldLow != null && shooterHeldAuto != null) {
       // Auto Aim Shot
-      buttons[Constants.LBumper].whileHeld(
+      buttons[Constants.LTriggerButton].whileHeld(
           shooterHeldAuto.beforeStarting(
               () -> {
-                shooterSubsystem.setAimMode(Constants.AimModes.AUTO);
+                shooterSubsystem.setAimMode(Constants.AimModes.TARMAC);
               },
               shooterSubsystem));
       // Fender Shot
-      buttons[Constants.LTriggerButton].whileHeld(
+      buttons[Constants.LBumper].whileHeld(
```

Comment:
According to the team discussion on 2/17, left trigger is meant to be the fender shot and left bumper is meant to be auto-shoot. Did the drive team request a change?

Reply:
Yeah ben wanted it changed, also auto-shoot probably wont be necessary for first competition

Reply:
When we have button change requests, we should track them somewhere, probably in #95?

Reply:
Got it